myFitnessTrainer/src/screens/AvatarScreen.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from 'react';
+import { Button, View, StyleSheet, TouchableOpacity, Text } from 'react-native';
+import * as ImagePicker from 'expo-image-picker';
+import { getStorage, ref, uploadBytes, getDownloadURL } from 'firebase/storage';
+import { storage } from '../../firebaseConfig';
+import Avatar from '../components/Avatar';
+
+const pickedImageFileName = "avatar.png";
```

Comment:
instead of making the `pickedImageFileName = "avatar.png"`, I would suggest making this edit:

```suggestion
const pickedImageFileName = Date.now() + ".jpg";
```

What this will do is make a new image in our Firebase storage with a unique name (the now Date). Currently, with the name set to `avatar.png`, when a user changes out their image, it will replace all users avatar.png file with the new one for all users who had uploaded an image.

Reply:
Great point! I keep forgetting that we're going to have multiple users